File: Swift/jewels-and-stones.swift
class Solution {
    func numJewelsInStones(_ J: String, _ S: String) -> Int {
        let setJ = Set(J)
        var res = 0

        for c in S {
            if setJ.contains(c) {
                res += 1
            }
        }

        return res
    }
}

File: Python/jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        setJ = set(J)
        res = 0

        for c in S:
            if c in setJ:
                res += 1

        return res
